pymilvus/client/stub.py

```diff
@@ -1068,3 +1068,43 @@ def get_replicas(self, collection_name: str, timeout=None, **kwargs) -> Replica:
         """
         with self._connection() as handler:
             return handler.get_replicas(collection_name, timeout, **kwargs)
+
+    def bulk_load(self, collection_name: str, partition_name: str, is_row_based: bool, files: list, timeout=None, **kwargs) -> list:
+        """ bulk load entities through files
+
+        :param collection_name: the name of the collection
+        :type  collection_name: str
+
+        :param partition_name: the name of the partition
+        :type  partition_name: str
+
+        :param is_row_based: indicate whether the files are row-based or coloumn based.
+        :type  is_row_based: bool
+
+        :param files: file names to bulk load
```

Comment:
Maybe we should point out that the file is located in `Milvus Storage`, not in local disk.